client/src/redux/auth.js

```diff
@@ -1,65 +1,85 @@
-import API from '../utils/API'
+import API from "../utils/API";
 // global state for authentification
 
 const initialState = {
-    email: "",
-    password: "",
-    errorMessage: "",
-    token: "",
-    authenticated: false,
-}
+	email: "",
+	password: "",
+	errorMessage: "",
+	token: "",
+	authenticated: false,
+};
+
+export const registerComplete = (data) => (dispatch) => {
+	dispatch({
+		type: "CLEAR_ERROR",
+	});
+	dispatch({
+		type: "LOGIN",
+		payload: data,
+	});
+};
+
 // set redux actions
 export const login = () => (dispatch, getState) => {
-    const {email, password} = getState().auth
-    const user = {email, password}
-    API.auth(user).then(response => {
-        dispatch({
-            type: "CLEAR_ERROR"
-        })
-        dispatch({
-            type: "LOGIN",
-            payload: response.data
-        })
-    })
-    .catch(error => {
-        dispatch({
-            type: "ADD_ERROR"
-        })
-    })
-}
+	const { email, password } = getState().auth;
+	const user = { email, password };
+	API.auth(user)
+		.then((response) => {
+			dispatch({
+				type: "CLEAR_ERROR",
+			});
+			dispatch({
+				type: "LOGIN",
+				payload: response.data,
+			});
+		})
+		.catch((error) => {
+			dispatch({
+				type: "ADD_ERROR",
+			});
+		});
+};
 
-export const logout = () => {
-    return {
-        type: "LOGOUT"
-    }
-}
+export const logout = () => async (dispatch) => {
+	await dispatch({
+		type: "LOGOUT",
+	});
+};
```

Comment:
was testing why my logout wasn't working.
don't really need to dispatch since this is only for async function